### ISSUE TITLE
Some crashes on mac 

### ISSUE THREAD

**Issue by dosipod** [jziolkowski/tdm#239]
![image](https://user-images.githubusercontent.com/87589371/231335154-0eaf53c1-92bf-4d91-bbe0-28b0db365858.png)
Can't open the app to get logs ..etc .

**Comment by Jason2866** [comment]
It does run on my Mac. Without telling which OS version running we can not help.

**Comment by jlk** [comment]
On MacOS 13.5.1 (Intel) I'm running into this as well. Currently running python3 version 3.11.4_1.

tdmgr installed via `pip3 install tdmgr`

### Use case to reproduce:

1. Run `tdmgr.py` from cli
2. Once UI comes up. select MQTT->Broker from menu bar and enter specs for your mqtt broker to connect to and click **Save** button (do not select _Connect on startup_ as this results on crash at startup until you delete `~/TDM/tdm.cfg`)
3. Attempt to connect to MQTT by clicking MQTT->Connect from menubar
4. App will crash with the following trace:
```
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/Util/models.py", line 281, in paint
    p.drawPixmap(option.rect.x() + 2, option.rect.y() + px_y, px.scaled(24, 24))
TypeError: arguments did not match any overloaded call:
  drawPixmap(self, QRectF, QPixmap, QRectF): argument 1 has unexpected type 'int'
  drawPixmap(self, QRect, QPixmap, QRect): argument 1 has unexpected type 'int'
  drawPixmap(self, Union[QPointF, QPoint], QPixmap): argument 1 has unexpected type 'int'
  drawPixmap(self, QPoint, QPixmap): argument 1 has unexpected type 'int'
  drawPixmap(self, QRect, QPixmap): argument 1 has unexpected type 'int'
  drawPixmap(self, int, int, QPixmap): argument 2 has unexpected type 'float'
  drawPixmap(self, int, int, int, int, QPixmap): argument 2 has unexpected type 'float'
  drawPixmap(self, int, int, int, int, QPixmap, int, int, int, int): argument 2 has unexpected type 'float'
  drawPixmap(self, int, int, QPixmap, int, int, int, int): argument 2 has unexpected type 'float'
  drawPixmap(self, Union[QPointF, QPoint], QPixmap, QRectF): argument 1 has unexpected type 'int'
  drawPixmap(self, QPoint, QPixmap, QRect): argument 1 has unexpected type 'int'
```

**Comment by PVerburg** [comment]
Works on MacOs on intel 12.6, can’t run 13 here.

**Comment by metawops** [comment]
Got the exact same problem like @jlk here on my Intel iMac running macOS 13.6.1.
Also installed via `pip3 install tdmgr`. 
Running Python 3.11.6

**Comment by jziolkowski** [comment]
pls clone the source and install manually the latest dev version

**Comment by xinuxnt** [comment]
cloned the source and installed manually but same errors here just after setup mqtt broker

**Comment by jziolkowski** [comment]
Fixed with #268
